tools/test-js-with-node

```diff
@@ -155,7 +173,21 @@ print('Starting node tests...')
 
 # If we got this far, we can run the tests!
 try:
-    ret = subprocess.check_call(command)
+    # This is duplicate code of:
+    #   `def run_parallel(lint_functions)` via commands.py
+    # A possible TODO: Refactor the function to work for both cases.
+    pids = []
+    for tests in sub_tests:
+        pid = os.fork()
```

Comment:
Is something wrong with `subprocess` (`.Popen`, for this case)?

Reply:
I'll update this, does `subprocess.Popen` just help with cross-platform compatibility?